ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -3,14 +3,156 @@
+        self.course_id = self.ocw_parser.master_json["short_url"]
+    
+    def tear_down(self):
+        """
+        Remove any generated files and null out local variables
+        """
+        destination_dir = "ocw_data_parser/test_json/destination_dir"
+        self.ocw_parser = None
+        if os.path.isdir(destination_dir):
+            shutil.rmtree(destination_dir)
+        self.course_id = None
+
+    # utils.py
+
+    def test_update_local_file_location(self):
+        """
+        Extract local course media, update the location of one of the files 
+        and then assert that the location has indeed changed
+        """
+        self.set_up(extract_media_locally=True)
+        if len(self.ocw_parser.master_json["course_files"]) > 0:
+            test_file = self.ocw_parser.master_json["course_files"][0]
+            original_location = test_file["file_location"]
+            update_file_location(self.ocw_parser.master_json, 'test_location', obj_uid=test_file["uid"])
+            self.assertNotEqual(original_location, self.ocw_parser.master_json["course_files"][0]["file_location"])
+            self.tear_down()
+        else:
+            self.tear_down()
+            self.fail("test course has no local media to test")
+
+    def test_update_foreign_file_location(self):
+        """
+        Extract foreign course media, update the location of one of the files 
+        and then assert that the location has indeed changed
+        """
+        self.set_up(extract_foreign_media_locally=True)
+        if len(self.ocw_parser.master_json["course_foreign_files"]) > 0:
+            test_file = self.ocw_parser.master_json["course_foreign_files"][0]
+            original_location = test_file["file_location"]
+            original_filename = test_file["link"].split("/")[-1]
+            update_file_location(self.ocw_parser.master_json, 'test_location/' + original_filename)
+            self.assertNotEqual(original_location, self.ocw_parser.master_json["course_foreign_files"][0]["file_location"])
+            self.tear_down()
+        else:
+            self.tear_down()
+            self.fail("test course has no foreign media to test")
+
+    def test_get_binary_data_none(self):
+        """
+        Find the first file without a datafield property and attempt to get the binary data from it
+        """
+        self.set_up()
+        if len(self.ocw_parser.master_json["course_files"]) > 0:
+            for media in self.ocw_parser.master_json["course_files"]:
+                if "_datafield_image" not in media and "_datafield_file" not in media:
+                    data = get_binary_data(media)
+                    self.assertIsNone(data)
+        else:
+            self.tear_down()
+            self.fail("test course has no local media to test")
+
+    def test_get_correct_path_none(self):
+        """
+        Test passing in invalid data to get_correct_path
+        """
+        get_correct_path(None)
```

Comment:
What are we asserting? If we are asserting that everything is fine, we should do `assert get_correct_path(None) is None` or something like that. If we want to assert an exception, we can do

    with pytest.raises(ExceptionClass):
        get_correct_path(None)